static/app/components/forms/fields/choiceMapperField.tsx

```diff
@@ -46,7 +48,10 @@ export interface ChoiceMapperProps extends DefaultProps {
   /**
    * Props forwarded to the add mapping dropdown.
    */
-  addDropdown: StaticDropdownAutoCompleteProps;
+  addDropdown: Omit<SingleSelectProps<string>, 'options'> & {
+    items: Array<SelectOption<string>>;
+    noResultsMessage?: string;
+  };
```

Comment:
I kept the same interface as before because we send the props for this from the backend:

https://github.com/getsentry/sentry/blob/6e27aa8e9217b2bac446b12ef913872597182b46/src/sentry/integrations/jira_server/integration.py#L360-L364

as far as I can see, those integrations are the only one that use this field. However, I can’t access them because I’m lacking permissions. I created a story, but is there any way to access how this now works in prod @JonasBa @evanpurkhiser ?

Reply:
I've checked out the PR and validated the change doesn't break anything.